### PR TITLE
Bad Message Wizard

### PR DESCRIPTION
**Pull request by AdamHawtin**
## Context
A nice wrapper around API calls to the exception manager service will help us deal with ops issues easier

## What has Changed
* Add bad message wizard (his name is Tim)
* Add config for exception manager

## How to test
Run with linked branches and exception manager.
Run the message wizard with `pipenv run python bad_message_wizard.py`
Test it with a variety of bad messages on a variety of queues

# Links
https://trello.com/c/2rkYNAMa/1266-exception-manager-v10

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[Exception Manager v1.0](https://trello.com/c/2rkYNAMa/1266-exception-manager-v10)

**Review by NickGrantX — APPROVED** [review]
Brilliant work. Love it.

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

File: config.py [hunk 1]
```diff
@@ -17,3 +17,6 @@ class Config:
     DB_USESSL = os.getenv('DB_USESSL', '')
     CASEAPI_HOST = os.getenv('CASEAPI_HOST', 'localhost')
     CASEAPI_PORT = os.getenv('CASEAPI_PORT', '8161')
+    EXCEPTIONMANAGER_HOST = os.getenv('EXCEPTIONMANAGER_HOST', 'localhost')
+    EXCEPTIONMANAGER_PORT = os.getenv('EXCEPTIONMANAGER_PORT', '8666')
```
Comment (hbrace) [inline config.py:21]: The environment variables in k8s are called:
EXCEPTION_MANAGER_SERVICE_HOST
EXCEPTION_MANAGER_SERVICE_PORT

**Review by hbrace — APPROVED** [review]

**Pull request merged**